web/Web/web/static/js/document-viewer.js

```diff
@@ -776,6 +776,7 @@ docView.prototype = {
           'ctrl_f_terms_input': $("#search_content").val(),
           'csrfmiddlewaretoken': options.csrfmiddlewaretoken,
           'page_title': document.title,
+          'len_docs': len_docs,
```

Comment:
So instead of a new parameter, we just use `parent.viewStack.length`

Let's rename `len_docs` to something more descriptive? Maybe `current_docview_stack_size`

Reply:
We just need to store `parent.viewStack.length` before the callback is run, because it modifies the stack.